README

```diff
@@ -0,0 +1,27 @@
+a. What is an issue?
+GitHub Issues are items you can create in a repository to plan, discuss and track work.
+
+b. What is a pull request?
+to copy your change into the git
+
+c. How do I open up a pull request?
+in vscode, after the change , commit and sync, and pull the request via github
+
+d. Give me a step by step guide on how to add someone to your repository.
+In the "Access" section of the sidebar, click Collaborators. 
+Click Add people. In the search field, start typing the name of person you want to invite, 
+then click a name in the list of matches. Click Add NAME to REPOSITORY. 
+The user will receive an email inviting them to the repository.
+
+e. What is the difference between git and GitHub?
+git stands for the way that coders how they track their difference versions or features.
+github is a website based repo hosting service.
```

Comment:
"Git is a distributed version control system that tracks changes in any set of computer files, usually used for coordinating work among programmers who are collaboratively developing source code during software development."